packages/web/src/scss/components/Card/_Card.scss

```diff
@@ -0,0 +1,152 @@
+    min-width: 0; // 3.
+    color: theme.$color;
+
+    @include breakpoint.up(map.get(theme.$breakpoints, tablet)) {
+        --#{tokens.$css-variable-prefix}card-padding: #{theme.$padding-tablet};
+    }
+}
+
+// 4., 5.
+.Card:has(.CardLink) :where(a:not(.CardLink), button, p, ul, ol, dl) {
+    position: relative;
+    z-index: 1;
+}
+
+.Card--vertical {
+    grid-template-rows: auto 1fr auto;
+    grid-template-areas:
+        'media'
+        'body'
+        'footer';
+}
+
+.Card--horizontal {
+    grid-template-columns: auto 1fr;
+    grid-template-rows: auto 1fr auto;
+    grid-template-areas:
+        'media logo'
+        'media body'
+        'media footer';
+}
+
+.Card--horizontalReversed {
+    grid-template-columns: 1fr auto;
+    grid-template-rows: auto 1fr auto;
+    grid-template-areas:
+        'logo media'
+        'body media'
+        'footer media';
+}
+
+:is(.Card--horizontal, .Card--horizontalReversed):has(.CardArtwork:not(:only-child), .CardMedia:not(:only-child)) {
+    column-gap: theme.$gap;
+}
+
+.Card--boxed {
+    border: theme.$border-width theme.$border-style theme.$border-color;
+    border-radius: theme.$border-radius;
+    background-color: theme.$background-color;
+
+    &:has(.CardLink) {
+        @media (hover: hover) {
+            transition: box-shadow theme.$transition-duration theme.$transition-timing;
+
+            &:hover {
+                box-shadow: theme.$box-shadow-state-hover;
+            }
+        }
+    }
+
+    &:not(:has(.CardMedia--expanded)) {
+        padding: var(--#{tokens.$css-variable-prefix}card-padding);
+    }
+}
+
+.Card--boxed.Card--vertical {
+    &:has(.CardMedia--expanded) {
+        grid-template-columns:
+            var(--#{tokens.$css-variable-prefix}card-padding)
+            1fr
+            var(--#{tokens.$css-variable-prefix}card-padding); // 2.
+
+        grid-template-rows:
+            var(--#{tokens.$css-variable-prefix}card-padding)
+            auto
+            1fr
+            auto;
+        grid-template-areas:
+            'media media media'
+            'media media media'
+            '. body .'
+            '. footer .';
+    }
+
+    &:has(.CardMedia:not(:only-child)) {
+        padding-bottom: var(--#{tokens.$css-variable-prefix}card-padding);
+    }
+}
+
+.Card--boxed.Card--horizontal {
+    &:has(.CardMedia--expanded) {
+        grid-template-columns: auto 1fr;
+        grid-template-rows:
+            var(--#{tokens.$css-variable-prefix}card-padding)
+            auto
+            1fr
+            auto
+            var(--#{tokens.$css-variable-prefix}card-padding); // 2.
+
+        grid-template-areas:
+            'media .'
+            'media logo'
+            'media body'
+            'media footer'
+            'media .';
+    }
+
+    &:has(.CardMedia:not(:only-child)) {
+        padding-right: var(--#{tokens.$css-variable-prefix}card-padding);
+    }
+}
+
+.Card--boxed.Card--horizontalReversed {
+    &:has(.CardMedia--expanded) {
```

Comment:
🙃 🔫

Reply:
I hope I understand this correctly: you have marked the repeating code, am I right?

Reply:
Agreed in person that no change is needed, it's just the selectors are a little bit complex to grasp.